iothub/device/src/Transport/Mqtt/MqttTransportSettings.cs

```diff
@@ -182,7 +182,7 @@ public bool CertificateRevocationCheck
         public bool CleanSession { get; set; }
 
         /// <summary>
-        /// The interval, in seconds, that the client establishes with the service, for sending keep-alive pings.
+        /// The interval, in seconds, that the client establishes with the service, for sending keep-alive pings over TCP.
         /// The default is 300 seconds.
```

Comment:
I know this was already there, but we should probably add an explanation here that even though this property value defaults to 300 seconds, that means that the default keep-alive is 300/4 = 75 seconds.

Reply:
This is so messed up, I'm not convinced we can make it much better than it is with documentation. I'd rather defer to v2.